rotkehlchen/data_import/importers/binance.py

```diff
@@ -298,7 +367,7 @@ def process_entry(
     ) -> None:
         amount = data['Change']
         asset = data['Coin']
-        category = AssetMovementCategory.DEPOSIT if data['Operation'] == 'Deposit' else AssetMovementCategory.WITHDRAWAL  # noqa: E501
+        category = AssetMovementCategory.WITHDRAWAL if data['Operation'] == 'Withdraw' else AssetMovementCategory.DEPOSIT  # noqa: E501
```

Comment:
Why this change? Isn't it the same, but you just changed the order of the if? If the else covers more stuff then this is tricky and needs to be explained. If not undo this diff.